### PR TITLE
Add serial console and grub file for Softlayer

### PR DESCRIPTION
**Pull request by sladebaumann**
When deploying Ursula to Softlayer virtual servers,
there is no /etc/default/grub file and there is no
serial console on the virtual servers. These changes
fix that problem and allow the Ursula deploy to proceed.

### CODE REVIEW

File: roles/common/tasks/kernel-tuning.yml [hunk 1]
```diff
@@ -22,10 +22,14 @@
...
     - apply-sysctl
 
+- stat: path=/etc/default/grub
+  register: grub_defaults
+
 - name: "Kernel boot parameters: disable quiet boot"
   lineinfile: dest=/etc/default/grub
               regexp="^GRUB_CMDLINE_LINUX_DEFAULT="
               line="GRUB_CMDLINE_LINUX_DEFAULT=\"\""
+  when: grub_defaults.stat.exists == True
```
Comment (j2sol) [inline roles/common/tasks/kernel-tuning.yml]: Use `when: grub_defaults.stat.exists|bool`

**Comment by j2sol** [comment]
ok to test

File: roles/common/tasks/main.yml [hunk 2]
```diff
@@ -68,6 +68,7 @@
 
 # Include serial console before kernel-tuning to build serial_console_cmdline
 - include: serial-console.yml tty={{ common.serial_console }}
+  when: common.serial_console.enabled
```
Comment (j2sol) [inline roles/common/tasks/main.yml]: This needs a `|bool` filter.

**Comment by sladebaumann** [comment]
@j2sol Have the jenkins tests been failling? I'm just wondering what the status of this pull request is.

**Comment by j2sol** [comment]
The PR is failing with:
```
TASK: [common | serial console init script] ***********************************
fatal: [173.247.105.15] => error while evaluating conditional: common.serial_console.enabled|bool
```
At that point in the role it does not like that conditional. Investigation is needed.

File: roles/common/tasks/main.yml (hunk 2 above)
Comment (sladebaumann) [inline roles/common/tasks/main.yml]: So looking back at this change, it seems to be the reason that the PR is failing. Could the ```|bool``` filter not be necessary in this instance?

**Comment by sladebaumann** [comment]
retest

**Comment by paulczar** [comment]
closing old PRs

**Pull request closed without merging**